### ISSUE TITLE
[Bug]: Contrast is not good at StudentWork component Profile hover

### ISSUE THREAD

**Issue by yassirog** [fdnd-agency/fdnd#72]
### Description

When i tested the contrast of the hover of the studentwork profile i saw it was purple background with "black" text. I couldn't read it good. I did a contrast test, because of that. 

https://github.com/fdnd-agency/fdnd/blob/4637b8d6f7b8dfb5c4c8f1b45dc584cccd0e2c6b/src/lib/molecules/StudentWork.svelte#L134-L137

![Image](https://github.com/user-attachments/assets/8c8524fd-6ab6-4068-a8c0-83326b99f24b)

It failed the test.

### Expected behavior

When you hover I expected, a good contrast. I almost couldn't read the text.

### Actual behavior

Bad contrast on the hover, because of this I couldn't read what it said.

### What device are you experiencing the problem on?

Laptop

### What operating system are you experiencing the problem on?

macOS

### What browsers are you experiencing the problem on?

Chrome
